lib/ui/platform_dispatcher.dart

```diff
@@ -1300,10 +1310,12 @@ class ViewConfiguration {
     this.padding = WindowPadding.zero,
     this.gestureSettings = const GestureSettings(),
     this.displayFeatures = const <DisplayFeature>[],
-  });
+  }) : assert(window == null || view == null),
+    _view = view ?? window;
 
   /// Copy this configuration with some fields replaced.
   ViewConfiguration copyWith({
+    FlutterView? view,
```

Comment:
Deprecate the `window` parameter in the copyWith functions as well? That one will also need to be removed at some point, right?

Reply:
https://github.com/flutter/engine/pull/38453/commits/5c2e0a9d2a7250ab23e45dccd573c684fbf0791c